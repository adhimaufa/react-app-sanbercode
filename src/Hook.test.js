import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Hooks from './Components/pageHooks';

test('hooks click', () => {
  const component = renderer.create(
     <Hooks />
  );

  let tree = component.toJSON;
  expect(tree).toMatchSnapshot();

//   component.getInstance().setCount();
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
});
