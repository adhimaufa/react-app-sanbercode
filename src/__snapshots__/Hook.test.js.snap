// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hooks click 1`] = `[Function]`;
